src/redismodule.h

```diff
@@ -619,6 +623,9 @@ static const RedisModuleEvent
 #define REDISMODULE_SUBEVENT_EVENTLOOP_AFTER_SLEEP 1
 #define _REDISMODULE_SUBEVENT_EVENTLOOP_NEXT 2
 
+#define REDISMODULE_SUBEVENT_KEY_DELETED 0
```

Comment:
I think the module can know that after right (when it will get the expired/evicted key space notification)? So, it can be a nice addition, but not a must.